src/components/Hero/Hero.tsx

```diff
@@ -1,13 +1,28 @@
 "use client";
 import HeroDesktop from "../../assets/images/HeroDesktop.jpg";
+import MobileThree from "../../assets/images/mobileThree.jpg";
 import Image from "next/image";
 import styles from "./Hero.module.scss";
 import { GiHamburgerMenu } from "react-icons/gi";
 import { IconContext } from "react-icons";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 
 export function Hero() {
   const ICON_SIZE = "40px";
+  const [mobile, setMobile] = useState<boolean>(false);
```

Comment:
this should be avoided and could cause the content to flicker on first render. media queries are preferred. see other comment below

Reply:
use media queries to hide the images based on size